components/listCardVideo/listCardVideo.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+// Components
+import CardVideo from '../cardVideo/cardVideo';
+
+// Styeles
+import styles from '../../styles/Home.module.css';
+
+const ListCardVideo = ({ cards }) => {
```

Comment:
```suggestion
const VideoCardList = ({ cards }) => {
```
`ListCardVideo` sounds more like an action, rather than a name that a component should have.